src/watson/models.py

```diff
@@ -70,6 +70,9 @@ class SearchEntry(models.Model):
 
     meta_encoded = models.TextField()
 
+    def _deserialize_meta(self):
```

Comment:
This needs to be implemented on the SearchAdapter, as extending the SearchEntry model isn't really an option for users.

so _deserialize_meta needs to be something like:

``` py
SearchEngine._created_engines[self.engine_slug].get_adapter(ContentType.objects.get_for_id(self.content_type_id).model_class()).deserialize_meta(self.meta_encoded)
```

And then add a deserialize_meta() method to SearchAdapter.